packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js

```diff
@@ -854,8 +854,14 @@ class EnumerationForm extends React.Component {
 			// checking if the value already exist
 			const valueExist = this.valueAlreadyExist(value, prevState);
 			const [validateAndAddAction, validateAction, abortAction] = prevState.headerInput;
+			// in this case, we could have the loading state that implied we have just one icon
+			if (!validateAction && !abortAction) {
+				// returning null in setState prevent re-rendering
+				return null;
```

Comment:
not explicitly described in the documentation, make hard for future work on this to find reference on how this work.

Reply:
or please add a link to https://reactjs.org/blog/2017/09/26/react-v16.0.html#breaking-changes
wich explain how and why this work this way